pkg/scheduler/apis/config/types_pluginargs.go

```diff
@@ -57,6 +57,10 @@ type NodeResourcesFitArgs struct {
 	// IgnoredResources is the list of resources that NodeResources fit filter
 	// should ignore.
 	IgnoredResources []string
+	// IgnoredResourceGroups defines the list of resource groups that NodeResources fit filter should ignore.
+	// e.g. if group is ["example.com"], it will ignore all resource names that begin
+	// with "example.com", such as "example.com/aaa" and "example.com/bbb".
+	IgnoredResourceGroups []string
```

Comment:
would `ignoredResources: ["example.com/*"]` be simpler?

Reply:
where `*` is only allowed at the end?